gcore/src/exec.rs

```diff
@@ -96,11 +102,11 @@ pub fn wait() -> ! {
 
 /// Resume previously paused message handling.
 ///
-/// If message has been paused using the [`wait`] function, then it is possible
-/// to continue its execution by calling this function. `waker_id` specifies a
-/// particular message to be taken out of the *waiting queue* and put into the
-/// *processing queue*. `gas_limit` is the gas value to be transferred from the
-/// current handling message to the waked one.
+/// If a message has been paused using the [`wait`] function, then it is
+/// possible to continue its execution by calling this function. [`waker_id`]
```

Comment:
`waker_id` is not a function so I don't see the reason to make a link to it.
```suggestion
/// possible to continue its execution by calling this function. `waker_id`
```